### PR TITLE
systemd-cgtop: add page

### PR DESCRIPTION
**Pull request by dadav**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

see #10191

### CODE REVIEW

File: pages/linux/systemd-cgtop.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# systemd-cgtop
+
+> Show the top control groups of the local Linux control group hierarchy, ordered by their CPU, memory, or disk I/O load.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-cgtop.html>.
+
+- Start an interactive view, similar to `top`:
```
Comment (kbdharun) [inline pages/linux/systemd-cgtop.md:6]:
```suggestion
- Start an interactive view:
```

File: pages/linux/systemd-cgtop.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# systemd-cgtop
+
+> Show the top control groups of the local Linux control group hierarchy, ordered by their CPU, memory, or disk I/O load.
```
Comment (kbdharun) [inline pages/linux/systemd-cgtop.md:3]:
```suggestion
> Show the top control groups of the local Linux control group hierarchy, ordered by their CPU, memory, or disk I/O load.
> See also: `top`.
```

File: pages/linux/systemd-cgtop.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-cgtop.html>.
+
+- Start an interactive view, similar to `top`:
+
+`systemd-cgtop`
+
+- Change the sort order to one of `cpu`, `memory`, `path`, `tasks` or `io`):
+
+`systemd-cgtop --order={{resource}}`
```
Comment (kbdharun) [inline pages/linux/systemd-cgtop.md:12]:
```suggestion
- Change the sort order:

`systemd-cgtop --order={{cpu|memory|path|tasks|io}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Pull request merged**